src/execution.ts

```diff
@@ -119,6 +121,7 @@ const GRAPHQL_ERROR = "__context.GraphQLError";
 const GLOBAL_RESOLVE = "__context.resolve";
 const GLOBAL_PARENT_NAME = "__parent";
 const LOCAL_JS_FIELD_NAME_PREFIX = "__field";
+const LOCAL_OBJECT_VAR_NAME = "__object";
```

Comment:
If we move to the undefined approach we will be able to remove this.

Reply:
done 👍